### PR TITLE
Lab meeting blog | 19 September 2019

### PR DESCRIPTION
**Pull request by KirstieJane**
Written quite quickly! Please check for typos 💖

### CODE REVIEW

**Review by yochannah — COMMENTED** [review]
I had a scan through this and didn't spot any typos!

File: _posts/blog/2019-09-19-2019-09-19-lab-meeting.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+---
+
+We didn't have a lab meeting today because the timing of the meeting coincided with the Turing's [Data Science for Mental Health Interest Group](https://turing-ds4mh.github.io/index.html).
+
+The two talks were excellent.
+[Neil Oxtoby](http://neiloxtoby.com/) presented an introduction to data-driven disease progression modelling, and [Alexandra Young](https://ayoungresearch.wordpress.com/) extended Neil's presentation with a specific example of data-driven modelling and progression of subtypes of dementia.
+
+This weekend - Saturday 21 September - is [World Alzheimer's Day](https://www.alzheimers.org.uk/get-involved/world-alzheimers-day) and while we waited we watched Samuel L Jackson explain how you - and an orange 🍊 - can eliminate the stigma and dispel the myth that there's nothing that can be done about dementia.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/L3AyD4RABOc" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (sgibson91) [inline _posts/blog/2019-09-19-2019-09-19-lab-meeting.md:23]: As on the other PR, this video doesn't render in Markdown but Jekyll may handle it for publishing.

**Review by sgibson91 — APPROVED** [review]

File: _posts/blog/2019-09-19-2019-09-19-lab-meeting.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+**Kirstie** is getting excited for Mozilla's annual festival: [www.mozillafestival.org](https://www.mozillafestival.org/en/) on 21 to 27 October.
+She didn't put in any session applications this year but would love for folks in the group to share what they're working on as part of the many many sessions on [openness](https://www.mozillafestival.org/en/spaces/openness/), [digital inclusion](https://www.mozillafestival.org/en/spaces/digital-inclusion/), [neurodiversity](https://www.mozillafestival.org/en/spaces/neurodiversity/) and [web literacy](https://www.mozillafestival.org/en/spaces/web-literacy/) 💖
+
+You can also support **Yo**'s sessions on ["Exhausted impostors and superstar masks"](https://public.zenkit.com/i/2RH604FcHf/piV4I8VtX/exhausted-impostors-and-superstar-masks?v=A2QcaWQ-QO) (with [Jez Cope](https://twitter.com/jezcope) and [Emmy Tsang](https://twitter.com/emmy_ft), and inspired by a [twitter thread](https://twitter.com/yoyehudi/status/1118474644766560256))and on ["Choosing an effective mentorship program"](https://public.zenkit.com/i/2RH604FcHf/sCleORy_8MO/choosing-an-effective-mentorship-program?v=A2QcaWQ-QO) (with **Malvika**).
```
Comment (LouiseABowler) [inline _posts/blog/2019-09-19-2019-09-19-lab-meeting.md]:
```suggestion
You can also support **Yo**'s sessions on ["Exhausted impostors and superstar masks"](https://public.zenkit.com/i/2RH604FcHf/piV4I8VtX/exhausted-impostors-and-superstar-masks?v=A2QcaWQ-QO) (with [Jez Cope](https://twitter.com/jezcope) and [Emmy Tsang](https://twitter.com/emmy_ft), and inspired by a [twitter thread](https://twitter.com/yoyehudi/status/1118474644766560256)) and on ["Choosing an effective mentorship program"](https://public.zenkit.com/i/2RH604FcHf/sCleORy_8MO/choosing-an-effective-mentorship-program?v=A2QcaWQ-QO) (with **Malvika**).
```

Comment (LouiseABowler) [reply _posts/blog/2019-09-19-2019-09-19-lab-meeting.md]: The change isn't showing clearly on here, but it's the tiniest of tiny edits! Needed a space between "twitter thread" and "and".

**Review by LouiseABowler — APPROVED** [review]

File: _posts/blog/2019-09-19-2019-09-19-lab-meeting.md (hunk 1 above)
Comment (KirstieJane) [reply _posts/blog/2019-09-19-2019-09-19-lab-meeting.md:23]: Thanks @sgibson91 - looks ok when rendered but I appreciate you checking

File: _posts/blog/2019-09-19-2019-09-19-lab-meeting.md (hunk 2 above)
Comment (KirstieJane) [reply _posts/blog/2019-09-19-2019-09-19-lab-meeting.md]: You're the best @LouiseABowler - thank you!

**Review by minnieho1115 — APPROVED** [review]
I had read all of this,I think everything looks great.

**Pull request merged**